Perlang.Common/Stmt.cs

```diff
@@ -4,6 +4,7 @@
 // Instead, change the ./scripts/generate_ast_classes.rb script that generated this code.
 //
```

Comment:
Note: this is no longer the case. We should remove `scripts/generate_ast_classes.rb` now that we are grown out of it, and remove this comment.

Reply:
Fixed, resolving.